index.js

```diff
@@ -0,0 +1,43 @@
+var express = require('express');
+var app = express();
+var metrics = require('node-monitor-ui');
+
```

Comment:
Please add a way to switch off the automatic garbage collector (via an environment variable).

Reply:
I will look for this possibility, by the research it is not possible to disable the GC of the NodeJs, what is possible is to change the parameters in a way that is never executed

https://github.com/nodejs/help/issues/462